eng/pipelines/templates/jobs/ci.yml

```diff
@@ -190,6 +190,40 @@ jobs:
           failOnStderr: false
       - template: /eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml
 
+  - job: "ComplianceTools"
+    pool:
+      name: azsdk-pool-mms-win-2019-general
+      vmImage: MMS2019
+    steps:      
+      - pwsh: |
+          New-Item -Path "$(Build.sourcesdirectory)" -Name "changedFiles" -ItemType "directory"
+          Write-Host "$(System.PullRequest.TargetBranch)"
+          git diff "$(System.PullRequest.TargetBranch)" HEAD --name-only | ForEach-Object {Copy-Item -Path $_ -Destination "$(Build.sourcesdirectory)/changedFiles"}
```

Comment:
I don't think we should copy the files but instead just find the directory they are in and add that directory to the list.

Reply:
Then we will scan the entire folder even if we only touch some of the files. It is not the fastest way we can do.

Reply:
Copying the files will cause all sorts of other potential issues so I'd like to avoid that if we can.